modules/core/src/main/java/org/apache/ignite/internal/processors/metastorage/persistence/DistributedMetaStorageImpl.java

```diff
@@ -279,6 +279,8 @@ public DistributedMetaStorageImpl(GridKernalContext ctx) {
             DistributedMetaStorageUpdateAckMessage.class,
             this::onAckMessage
         );
+
+        ctx.internalSubscriptionProcessor().registerGlobalStateListener(this);
```

Comment:
Let's add `// No-op.` in `onActivate(GridKernalContext kctx)`

Reply:
Fixed